README.md

```diff
@@ -13,6 +13,30 @@ This gives you the power to prioritize our work and support the project contribu
 # Getting started
 ---
 
+### If you are using `react-native` >= 0.60.0 please use `react-native-share` >= 2.0.0
+
+## Automatic Way
+---
+``` 
+yarn add react-native-share
+```
+
+or if you're using npm
+``` 
+npm install react-native-share --save
+```
+---
+
+#### Important:
+Linking is not needed anymore. ``react-native@0.60.0+`` supports dependencies auto linking.
+For iOS you also need additional step to install auto linked Pods (Cocoapods should be installed):
+``` 
+cd ios && pod install && cd ../
+```
+___
+
+### If you are using `react-native` <= 0.59.10 please use `react-native-share` <= 1.2.1:
+
```

Comment:
Would be interesting warning about the possible problems using this PR. Adding something like: 

```suggestion
If you are having any problems with this library, or need to use >= 2.0.0 please refer to: [jetifier](https://github.com/mikehardy/jetifier#to-reverse-jetify--convert-node_modules-dependencies-to-support-libraries).

After installing jetifier, runs a ```npx jetify -r``` and test if this works by running a ```react-native run-android```.
```